lib/ssh_scan/sqlite_db.rb

```diff
@@ -0,0 +1,64 @@
+require 'sqlite3'
+
+module   SSHScan
+  module Database
+    class SQLiteDb
```

Comment:
Ideally, put sqlite_db.rb and mongo_db.rb in a directory called "database", then in database.rb, include both these files.